apps/public-docsite-resources/src/components/pages/PivotPage.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
```

Comment:
Since we're updating file names here should we also rename this to `TabsPage`? Similar comment for other files below.